core/src/commonMain/kotlin/com/algolia/instantsearch/core/tree/Node.kt

```diff
@@ -1,7 +1,17 @@
 package com.algolia.instantsearch.core.tree
 
-
 public data class Node<T>(
     val content: T,
     val children: MutableList<Node<T>> = mutableListOf()
-)
+) {
+
+    public var selected: Boolean = false
+
+    constructor(
+        content: T,
+        selected: Boolean,
```

Comment:
I would say we should use the same naming as above: `isSelected` instead of `selected`